tests/typecheck/core/test_checks.yml

```diff
@@ -27,3 +27,45 @@
 
     @register()  # E: Value of type variable "_C" of function cannot be "Callable[[int], Sequence[CheckMessage]]"  [type-var]
     def wrong_args(bla: int) -> Sequence[CheckMessage]: ...
+- case: test_checks_register_as_function
```

Comment:
Thanks! But tests for this feature is not required :)

Reply:
Does that mean I should remove the new tests before this PR is accepted?

Reply:
Yes, please :)